### ISSUE TITLE
Installation instructions are missing crucial context

### ISSUE THREAD

**Issue by dhdaines** [jorgenschaefer/elpy#1998]
The installation instructions say to add `use-package` to your `.emacs`.  This neglects to mention that **before** you do this, you actually have to have:

- Enabled MELPA
- Run `package-refresh-contents`
- Installed `use-package`

Otherwise, you will get all manner of cryptic Lispy error messages.

At least on Ubuntu 22.04, these things are not done for you when you install Emacs.

**Comment by sboisen** [comment]
Thanks, i hit the same problem, and this solved it (MacOS).

**Comment by gopar** [comment]
Right under the installation code snippet with `use-package` there is a link to other methods of installation which show the melpa way. I do agree that perhaps showing `use-package` as the default installation method might not be the best for those who don't know about the package.

**Comment by dhdaines** [comment]
Yes, but the issue is that in order to use `use-package`, you have to first *install* `use-package` with MELPA, and unless you know this already, well.. you don't know this already.
